src/z3c/macro/tales.py

```diff
@@ -48,7 +49,7 @@ def __call__(self, econtext):
     from chameleon.codegen import template
 
     class MacroGetter(object):
-        """Collect named IMacroTemplate via a TAL namespace called ``macro``."""
+        """Collect named IMacroTemplate via a TAL namespace called `macro`."""
```

Comment:
Double-backticks is ReST for monospace text.

Single backticks is ReST for unspecified configurable object references.

I'm not sure this change is right.